### PR TITLE
Weekly Update - Aug 22, 2024

### PR DESCRIPTION
**Pull request by justcabyr**

### CODE REVIEW

File: _data/jobs.yml [hunk 1]
```diff
@@ -72,6 +76,12 @@
   jobs: []
 - company: nlc
   jobs: []
+- company: keen
+  jobs:
```
Comment (jackharrhy) [inline _data/jobs.yml:80]: looks like this has to be added in companies.yml, build is failing

**Review by jackharrhy — DISMISSED** [review]

File: _data/jobs.yml (hunk 1 above)
Comment (justcabyr) [reply _data/jobs.yml:80]: Thanks, seen. First time adding jobs.

**Comment by MitMaro** [comment]
I pushed the generated changes, since @justcabyr 's laptop didn't have the `yaml` pacakge.

**Comment by MitMaro** [comment]
Thanks @justcabyr !

**Review by MitMaro — APPROVED** [review]

**Pull request merged**